youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+msgstr "最低でも1つはURLを入力する必要があります"
+
+#: youtube_dl_gui/mainframe.py:124
+msgid "Downloads started"
+msgstr "ダウンロードを開始しました"
+
+#: youtube_dl_gui/mainframe.py:125
+msgid "Choose Directory"
+msgstr "フォルダを選択してください"
+
+#: youtube_dl_gui/mainframe.py:127
+msgid "Download in progress. Please wait for all downloads to complete"
+msgstr "ダウンロード中です。すべて完了するまでお待ちください。"
+
+#: youtube_dl_gui/mainframe.py:128
+msgid "Update already in progress"
+msgstr "更新中です"
+
+#: youtube_dl_gui/mainframe.py:130
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "youtube-dlの最新版をダウンロードしています。お待ちください..."
+
+#: youtube_dl_gui/mainframe.py:131
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Youtube-dlのダウンロードに失敗しました [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "Youtube-dlのダウンロードに成功しました"
+
+#: youtube_dl_gui/mainframe.py:134
+msgid "Unable to open directory: '{dir}'. The specified path does not exist"
+msgstr "フォルダを開くことができません: '{dir}'。指定されたパスが存在しません。"
+
+#: youtube_dl_gui/mainframe.py:136
+msgid "Error while shutting down. Make sure you typed the correct password"
+msgstr "シャットダウンの際にエラーが発生しました。必ず正確なパスワードを入力したことを確認してください"
+
+#: youtube_dl_gui/mainframe.py:138
+msgid "Shutting down system"
+msgstr "システムをシャットダウンします"
+
+#: youtube_dl_gui/mainframe.py:141
+msgid "Extension"
+msgstr "拡張子"
+
+#: youtube_dl_gui/mainframe.py:142
+msgid "Size"
+msgstr "サイズ"
+
+#: youtube_dl_gui/mainframe.py:143
+msgid "Percent"
+msgstr "パーセント"
+
+#: youtube_dl_gui/mainframe.py:144
+msgid "ETA"
+msgstr "ETA"
+
+#: youtube_dl_gui/mainframe.py:145
+msgid "Speed"
+msgstr "速度"
+
+#: youtube_dl_gui/mainframe.py:146
+msgid "Status"
+msgstr "状態"
+
+#: youtube_dl_gui/mainframe.py:235
+msgid "Get URL"
+msgstr "URLをコピー"
+
+#: youtube_dl_gui/mainframe.py:236
+msgid "Get command"
+msgstr "コマンドをコピー"
+
+#: youtube_dl_gui/mainframe.py:237
+msgid "Open destination"
+msgstr "保存先フォルダを開く"
+
+#: youtube_dl_gui/mainframe.py:238
+msgid "Re-enter"
+msgstr "再登録"
+
+#: youtube_dl_gui/mainframe.py:458
+msgid "Resume"
+msgstr "再開"
+
+#: youtube_dl_gui/mainframe.py:480
+msgid "Video"
+msgstr "動画"
+
+#: youtube_dl_gui/mainframe.py:484
+msgid "Audio"
+msgstr "音声"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "No items selected. Please pick an action"
+msgstr "項目が選択されていません。操作項目を選択してください"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove all"
+msgstr "全消去"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove completed"
+msgstr "消去完了"
+
+#: youtube_dl_gui/mainframe.py:534
+msgid "Are you sure you want to remove selected items?"
+msgstr "選択アイテムを消去してよろしいですか?"
+
+#: youtube_dl_gui/mainframe.py:546
+msgid "Item is active, cannot remove"
+msgstr "使用中のため削除できません"
+
+#: youtube_dl_gui/mainframe.py:579
+msgid "Item is not completed"
+msgstr "未完了のアイテムです"
+
+#: youtube_dl_gui/mainframe.py:668
+msgid "Update in progress. Please wait for the update to complete"
+msgstr "アップデート中です。完了までお待ちください"
+
+#: youtube_dl_gui/mainframe.py:716
+msgid "Logging is disabled"
+msgstr "ロギング無効"
```

Comment:
According to Google the msgstr field translates to "Invalid logging".
Are you sure this is correct?

Reply:
It's OK in Japanese.